demo/oc_connection/molecule/default/templates/test.yml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "test1"
+  namespace: "jeffcpullen-devspaces"
```

Comment:
Is there a way to not link this to your user? / peeking around I don't see this file being used? I might be missing it (?)

Reply:
extra file, don't need. removed